### DISCUSSION TITLE
How can I add the "Skill' from the People card/Delve as a filter?

### DISCUSSION THREAD

**Discussion by AdamSzabo0721** [microsoft-search/pnp-modern-search#3689] [Q&A] [answered]
Hi,

I'm trying to create a people search, and would like to be able to filter on the skills from people's profile. 
I cant seem to find any useful information regarding.

Does it has to be made via a refinable string?

**Comment by wobba** [comment, marked as answer]
Any filter in the refinement web part has to be populated into a refinable property. `Skills` is not refinable, so you need to map `People:SPS-Skills` to a new property and re-index all the profiles. You may try to use `PeopleKeywords` instead and see if this works - but it has data from interests and responsibility as well.

**Reply by AdamSzabo0721** [reply]
Thank you for the great feedback, this is indeed the solution. Being able to search on skills(as it's a Managed Property), and being able to add it to the filter web part, confused me and I thought I just miss here something. But indeed I checked and the skills property itself is not refinable out of the box. 
I find PeopleKeywords great as I would need exactly those information(Interests, Responsibilities, Skills). Is there anyway I to use the PeopleKeywords but still be able to filter separately on skills, interests, etc.? 
So I would need to remap these skills to separate refinable strings.
